src_tests/manualtests/osm_configurator/view/controller_stub/settings_controller_stub.py

```diff
@@ -4,8 +4,14 @@
 
 
 class SettingsControllerStub(ISettingsController):
+    def get_number_of_key_recommendations(self) -> int:
+        return 0
+
+    def set_number_of_key_recommendations(self, number_of_processes: int) -> bool:
+        return False
+
     def get_number_of_processes(self) -> int:
```

Comment:
If test behavior is fixed it's better to skip the tests i guess. Please use the pytest decorator to skip the test